src/msg.rs

```diff
@@ -19,7 +19,7 @@ pub struct InitMsg {
     pub symbol: String,
     pub decimals: u8,
     pub initial_balances: Vec<InitialBalance>,
-    pub prng_seed: String,
+    pub prng_seed: Binary,
```

Comment:
That means that `prng_seed` has to be a valid base64 string right? Isn't it kinda limits the entropy of the RNG?

Reply:
No, since you can encode any binary slice as base64. That's why the type here is called Binary, and wraps a `Vec<u8>`. should i fix up any tests though?